### PR TITLE
BAM-1910 adding legal docs

### PR DESCRIPTION
**Pull request by PhilWinchester**

### CODE REVIEW

File: docs/dataretentionpolicy.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
+Data Retention Policy
+=====================
+
+1. Overview
+-----------
+
+Narrativ generates and encounters various forms of data in operating our platform.  This includes log data, operational data and user data.   All data has some value, whether for troubleshooting or monitoring existing systems, analysis for improvement of future systems, or reporting past results of platform operations. However, much of it has the potential to contain sensitive information and must be deleted to comply with our data protection policies and the law.
```
Comment (myfashionhub) [inline docs/dataretentionpolicy.rst]: Too many spaces before `All data has some value`

File: docs/privacypolicy.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Publishers and Merchants and other clients may also have their own policies that govern how they collect, use, and share data. These policies may differ from Narrativ’s policies described in this Privacy Policy. Please consult the privacy policies of the websites you visit and apps you use to become familiar with their privacy practices and to learn about any choices that these companies may offer with respect to their information practices. In addition, any website containing our Services may contain links to websites or content operated and maintained by third parties, over which we have no control. We encourage you to review the privacy policy of a third-party website before disclosing any information to the website.
+
+Information Collection and Use
+------------------------------
+
+Narrativ collects data in a variety of ways - including through the use of log files, pixel tags, cookies, and/or similar technologies. Examples of the types of data that we collect are:
+
+  * Browser information (e.g. URL, browser type, ‘click through’ data)
```
Comment (myfashionhub) [inline docs/privacypolicy.rst]: double quotes?

File: docs/dataretentionpolicy.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  * only contain personal information (such as names and email addresses) to the extent required for application functionality;
+
+  * be protected with password; and
+
+  * have user account access limited to employees with a legitimate need of use.
+
+Backup operational data should be maintained for no longer than 45 days, and protected with the same level of security as the original data.
+
+
```
Comment (xak200) [inline docs/dataretentionpolicy.rst]: there are 2 newlines here where there should only be one

File: docs/privacypolicy.rst (hunk 2 above)
Comment (PhilWinchester) [reply docs/privacypolicy.rst]: I copied this straight from the google doc we are using now. I'm going to defer to the original writer.

**Pull request merged**